fbpic/boundaries/boundary_communicator.py

```diff
@@ -792,7 +799,7 @@ def exchange_particles_aperiodic_subdomain(self, species, fld, time ):
     def damp_EB_open_boundary( self, interp ):
```

Comment:
There is a bug in line 855. You pass self.d_left_damp instead of self.d_right_damp to the GPU kernel. Apparently I missed this in the previous PR.